build/scripts/build_helper.fb_folly

```diff
@@ -132,6 +132,10 @@ install_fb_folly_from_source(){
     else
         git checkout -f v2019.11.11.00
     fi
+    if [[ "$(get_distribution_release)" == "ubuntu22.04" ]]; then
+       sed -i -e "s@#include <algorithm>@#include <algorithm>\n#include <limits>@" folly/TimeoutQueue.cpp
+       sed -i -e "s@folly::constexpr_max@@" -e "s@SIGSTKSZ, @@" ./folly/fibers/FiberManager.cpp
+    fi
```

Comment:
@NeptuneMagicSauce (romain)

I would do this patch all the time. I know we have to do it also for `fedora-38`.

Reply:
Will this be backward-compatible? Will it work for Ubuntus 18 and 20?

Reply:
yes